addons/mail/tests/test_translation_controller.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.tests import HttpCase
+from unittest.mock import patch
+import requests
+import json
+
+
+SAMPLE = {
+    "text": "Al mal tiempo, buena cara.",
+    "src": "es",
+    "lang": "Spanish",
+    "en": "To bad weather, good face.",
+    "fr": "Au mauvais temps, bonne tête.",
+}
+
+
+def mock_response(fun):
+    def wrapper(self, url, data=False, params=False):
+        response = requests.Response()
+        response.status_code = 200
```

Comment:
note that for HTTP status codes, you could use enums from the [http package](https://docs.python.org/3/library/http.html) for more transparency:
```suggestion
        response.status_code = HTTPStatus.OK
```